public/about/about.pug

```diff
@@ -1,30 +1,67 @@
+      p Reproduction of material under license CC BY-NC-ND 4.0 must meet the following requirements:
+      p XXXXXX
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") By - Include the Following Attribution:
+    .text-serif(style="text-align: left; color: black;")
+      p Reprinted from The Grand Tour Explorer, created by Giovanna Ceserani and Giorgio Caviglia, Part 3 of Giovanna Ceserani, 
+        i A World Made by Travel
+        | , 
+        a(href="https://www.worldmadebytravel.org/thegrandtourexplorer") https://www.worldmadebytravel.org/thegrandtourexplorer
+        | , published by Center for Spatial and Textual Analysis (CESTA) 0 Stanford 2019. All rights reserved. Licensed under the Creative Commons License CC BY-NC-ND 4.0, 
+        a(href="https://creativecommons.org/licenses/by-nc-nd/4.0/") https://creativecommons.org/licenses/by-nc-nd/4.0/
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") NC - Non-commercial Purpose or Use:
+    .text-serif(style="text-align: left; color: black;")
+      p The content or any portion of this website cannot be sold, redistributed, linked, and/or licensed for money, used for promotional purpose, and/or included in works available for sale or licensing, even by non-profit entities.
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") ND - No-derivative:
+    .text-serif(style="text-align: left; color: black;")      
+      p If you remix, transform, or build upon the material, you may not distribute the modified material.
+  
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Permission
+    .text-serif(style="text-align: left; color: black;")      
+      p Third party material: Permission to use third party illustrations and/or maps included in this website should be requested directly from their respective sources.
+      p Commercial and/or derivative use: Please contact Stanford University Press if you have any questions regarding the reproduction of material included in this website and/or for all requests that don't meet the Creative Commons License CC BY-NC-ND 4.0 requirements.
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Links
+    .text-serif(style="text-align: left; color: black;")      
+      p Links from this website are provided for information and convenience only and we assume no responsibility or liability for the availability or content of any linked third party website. We do not endorse any linked website.
   
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Platform
+    .text-serif(style="text-align: left; color: black;")
+      p '
+        i The Grand Tour Explorer
+        | ' is a custom web application built using Angular JS, Python, D3 at CESTA.
```

Comment:
Can you include that the code is available at https://github.com/cestastanford/grandtour?

```suggestion
        | ' is a custom web application built using Angular JS, D3, and Python at CESTA.
```